riscv/insns/ssrdp.h

```diff
@@ -0,0 +1,7 @@
+#include "zicfiss.h"
+require_extension(EXT_ZICFISS);
```

Comment:
Should not require extension here either.

Reply:
Done